client/my-sites/customer-home/style.scss

```diff
@@ -19,32 +19,6 @@
 	grid-column: $col-start / span $span;
 }
 
-.customer-home__heading {
-	display: flex;
-
-	@include breakpoint-deprecated( '<660px' ) {
-		padding-right: 16px;
-	}
-
-	.formatted-header {
-		margin-right: 12px;
-	}
-
-	.formatted-header__subtitle {
-		margin: 0;
```

Comment:
Removing this `margin: 0` rule has changed the padding a little. Here's a gif of me switching tabs between your change and production:

![May-14-2021 16-22-30](https://user-images.githubusercontent.com/1500769/118221270-64f6ea00-b4d1-11eb-9792-800e3635007f.gif)

I think if we reinstate the `customer-home__heading` wrapper div, and this `formatted-header__subtitle` rule, then it'll keep the spacing exactly as it was before. Looks like the all the rest is good to delete though 👍

Reply:
Oops, should be fixed now! :)